packages/bash/etc-bash.bashrc

```diff
@@ -22,3 +22,6 @@ if [ -x @TERMUX_PREFIX@/libexec/termux/command-not-found ]; then
 		@TERMUX_PREFIX@/libexec/termux/command-not-found "$1"
 	}
 fi
+
+# https://github.com/archlinux/svntogit-packages/blob/packages/bash/repos/core-x86_64/system.bashrc#L22
```

Comment:
I don't think the "citation" is necessary. Besides the line number could change.

Reply:
OK, I'll remove it.